<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountsWindow</class>
 <widget class="QMainWindow" name="AccountsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accounts</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="accountsList"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addAccount">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset theme="list-add">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeAccount">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset theme="list-remove">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
